app/views/root/_stylesheet.html.erb

```diff
@@ -1,7 +1,7 @@
 <%
   css_file = local_assigns[:css_file] || 'static'
 %>
-<!--[if gt IE 8]><!--><%= stylesheet_link_tag css_file, integrity: false %><!--<![endif]-->
-<!--[if IE 8]><%= stylesheet_link_tag "#{css_file}-ie8" %><script>var ieVersion = 8;</script><![endif]-->
+<%= stylesheet_link_tag css_file, integrity: false %>
+<!--[if lt IE 9]><%= stylesheet_link_tag "#{css_file}-ie8" %><script>var ieVersion = 8;</script><![endif]-->
```

Comment:
Saying that if _less than_ IE 9 then `ieVersion` _is_ 8 feels slightly strange - it's the truth from a certain point of view.

What do we gain from setting this to be less than IE 9 rather than equal to IE 8?

(My other thought was digging through where this global variable is used and seeing if we can remove it - but that's a topic for another pull request.)

Reply:
Sorry, this was me being lazy - it should be `less than or equal to IE8` but I tried what I thought the syntax for that was and it didn't work, but the same file already had `less than IE9` in it, so I used that instead. I'll see if I can make it more explicit.

Reply:
Fixed.